app/controllers/topical_events_controller.rb

```diff
@@ -1,6 +1,15 @@
 class TopicalEventsController < ApplicationController
   def show
     @topical_event = TopicalEvent.find!(request.path)
+    @travel_advice = []
```

Comment:
I wonder if this might be better as a method on the `TopicalEvent` model (e.g. a `travel_advice` method), to keep the logic of getting content (even if it's hardcoded) separate from the controller?  Then in the view, it can be retrieved using `@topcial_event.travel_advice`.

Reply:
yeah good point. Have moved this over